io/net/ethtool_test.py

```diff
@@ -131,14 +135,31 @@ def test_ethtool(self):
         for state, status in zip(["down", "up"], ["no", "yes"]):
             if not self.interface_state_change(self.iface, state, status):
                 self.fail("interface %s failed" % state)
-            cmd = "ethtool %s %s %s" % (self.args, self.iface, self.elapse)
-            ret = process.run(cmd, shell=True, verbose=True,
-                              ignore_status=True)
-            if ret.exit_status != 0:
-                if "Operation not supported" in ret.stderr_text:
-                    self.log.warn("%s failed" % self.args)
-                else:
-                    self.fail("failed")
+            if self.args=="-L" :
+                if self.other!='' and self.combined!='' :
+                    cmd1 = "ethtool %s %s tx %s rx %s other %s combined %s" % (self.args, self.iface, self.tx, self.rx, self.other, self.combined)
+                else :
+                    cmd1 = "ethtool %s %s tx %s rx %s" % (self.args, self.iface, self.tx, self.rx)
```

Comment:
@TasmiyaNalatwad please take care of following pep8 warns,
test.py:138:25: E225 missing whitespace around operator
test.py:138:31: E203 whitespace before ':'
test.py:139:30: E225 missing whitespace around operator
test.py:139:52: E225 missing whitespace around operator
test.py:139:56: E203 whitespace before ':'
test.py:140:80: E501 line too long (146 > 79 characters)
test.py:141:21: E203 whitespace before ':'
test.py:142:80: E501 line too long (98 > 79 characters)
test.py:144:35: E128 continuation line under-indented for visual indent
test.py:150:35: E128 continuation line under-indented for visual indent
test.py:152:46: E225 missing whitespace around operator
test.py:152:80: E225 missing whitespace around operator
test.py:152:80: E501 line too long (83 > 79 characters)

Reply:
@TasmiyaNalatwad run pycodestyle & pylint  before pushing the PR, this will help to take care of your indentation issues.

Reply:
Taken care of Indentation error by running pycodestyle , Still 2 lines are exceeding 79 characters, But those are comands that we are executing to set Channel. Hence cannot modify those